cfme/containers/provider/openshift.py

```diff
@@ -14,7 +14,8 @@ def __init__(self, name, value, field_type=None, href=None):
 
 @ContainersProvider.add_provider_type
 class OpenshiftProvider(ContainersProvider):
-    STATS_TO_MATCH = ContainersProvider.STATS_TO_MATCH + ['num_route']
+    num_route_template = ['num_route'] + ['num_template']
+    STATS_TO_MATCH = ContainersProvider.STATS_TO_MATCH + num_route_template
```

Comment:
Happy to leave it but it doesn't seem necessary to create a new variable for num_route_template here